src/commands/jobs/util.ts

```diff
@@ -263,9 +263,9 @@ export const itemOutputFormat = {
 				if (!val) {
 					return '';
 				}
-				const size = !Boolean(val.source_to_destination)
-					? (val.source && val.source.bytes)
-					: (val.destination && val.destination.bytes);
+				const size = Boolean(val.source_to_destination) && val.destination && val.destination.bytes
```

Comment:
I think that this statement should be included within the `Boolean` expression.

Reply:
will do!